src/nix/nix.md

```diff
@@ -177,9 +184,17 @@ operate are determined as follows:
   /nix/store/q6580lr01jpcsqs4r5arlh4ki2c1m9rv-glibc-2.33-123-dev             44200560
   ```
 
-* If you didn't specify the desired outputs, but the derivation has an
-  attribute `meta.outputsToInstall`, Nix will use those outputs. For
-  example, since the package `nixpkgs#libxml2` has this attribute:
+  and likewise, again using a store path to a "drv" file to specify the derivation:
+
+  ```console
+  # nix path-info -S --eval-store auto --store https://cache.nixos.org '/nix/store/gzaflydcr6sb3567hap9q6srzx8ggdgg-glibc-2.33-78.drv^*'
+  …
+  ```
+
+* If you didn't specify the desired outputs, but the derivation comes
+  from an expression which has an attribute `meta.outputsToInstall`, Nix
```

Comment:
Probably best to change back to the original wording since a store .drv does not have `meta.outputsToInstall`.